lib/fuzzer.c

```diff
@@ -659,6 +664,10 @@ uint32_t fuzi_q_fuzzer(void* fuzz_ctx, picoquic_cnx_t* cnx,
             uint64_t next_step = fuzz_pilot & 0x03;
             size_t final_pad = length_non_padded(bytes, length, header_length);
             size_t fuzz_frame_id = (size_t)((fuzz_pilot >> 3) % nb_fuzi_q_frame_list);
+            
+            // Fuzzing frames based on the choice provided by the user
+            if (frame_fuzz_type!=INT_MIN)	fuzz_frame_id=fuzz_frame_id;
+
```

Comment:
For style consistency, please bracket the "then" block, as in
~~~
    if (frame_fuzz_type != INT_MIN){
        fuzz_frame_id=fuzz_frame_id;
    }
~~~
Also, please use spaces, not tabs.